newswires/conf/logback.xml

```diff
@@ -1,50 +1,50 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-
 <!-- https://www.playframework.com/documentation/latest/SettingsLogger -->
-
-<!DOCTYPE configuration>
-
 <configuration>
-  <import class="ch.qos.logback.classic.encoder.PatternLayoutEncoder"/>
-  <import class="ch.qos.logback.classic.AsyncAppender"/>
-  <import class="ch.qos.logback.core.FileAppender"/>
-  <import class="ch.qos.logback.core.ConsoleAppender"/>
 
-  <appender name="FILE" class="FileAppender">
+  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
     <file>${application.home:-.}/logs/application.log</file>
-    <encoder class="PatternLayoutEncoder">
+    <encoder>
       <charset>UTF-8</charset>
-      <pattern>%d{yyyy-MM-dd HH:mm:ss} %highlight(%-5level) %cyan(%logger{36}) %magenta(%X{pekkoSource}) %msg%n</pattern>
+      <pattern>
+        %d{yyyy-MM-dd HH:mm:ss} - [%level] - from %logger with markers=%marker %n%message%n%xException%n
+      </pattern>
     </encoder>
   </appender>
 
-  <appender name="STDOUT" class="ConsoleAppender">
-    <!--
-         On Windows, enabling Jansi is recommended to benefit from color code interpretation on DOS command prompts,
-         which otherwise risk being sent ANSI escape sequences that they cannot interpret.
-         See https://logback.qos.ch/manual/layouts.html#coloring
-    -->
-    <!-- <withJansi>true</withJansi> -->
-    <encoder class="PatternLayoutEncoder">
-      <charset>UTF-8</charset>
-      <pattern>%d{yyyy-MM-dd HH:mm:ss} %highlight(%-5level) %cyan(%logger{36}) %magenta(%X{pekkoSource}) %msg%n</pattern>
-    </encoder>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <if condition='property("STAGE").contains("DEV")'>
```

Comment:
add the .sbtopts file to make this work https://github.com/guardian/crosswordv2/blob/main/.sbtopts (this condition turns _off_ json encoding for running locally, which is much more human readable)

Reply:
ty! https://github.com/guardian/newswires/pull/31/commits/93a41c8dba21516e4ca5a945a6155371e58d4d07